content/docs/04-clusters/01-public-cloud/03-azure.md

```diff
@@ -58,7 +58,7 @@ To create an Azure cloud account, we need:
 
 For existing cloud account go to `Edit` and toggle the `Connect Private Cloud Gateway` option to select the created gateway from the drop down menu.
 
-Once the above informations are provided to the cloud account creation wizard, click on **Disable Properties**, if the user do not want to have a network call from Palette console to the Azure account for linking the Azure networking details to Palette console.  
+Once the above information are provided to the cloud account creation wizard, click on **Disable Properties**, if the user do not want to have a network call from Palette console to the Azure account for linking the Azure networking details to Palette console.  

```

Comment:
I don't understand what is being said here. Also, we should address the user with "you" and not use a third-person voice such as "if the user."  But this paragraph needs work as it's confusing to read.